client/components/event-form.jsx

```diff
@@ -125,6 +151,8 @@ export default function EventForm() {
         setDescriptionError(true);
       } else if (attribute === 'time') {
         setTimeError(true);
+      } else if (attribute === 'email') {
+        setEmailError(true);
       } else if (attribute === 'date') {
         setDateError(true);
       } else if (attribute === 'destination') {
```

Comment:
I don't really understand what `true` means for an error. Message me what you're trying to accomplish and I'll help you refactor.